src/transforms/tokenizer.rs

```diff
@@ -80,7 +81,9 @@ impl Transform for Tokenizer {
             for ((name, conversion), value) in self.field_names.iter().zip(parse(value).into_iter())
             {
                 match conversion.convert(value.as_bytes().into()) {
-                    Ok(value) => event.as_mut_log().insert_explicit(name.clone(), value),
+                    Ok(value) => match value {
```

Comment:
this match statement seems redundant since you're matching on the catch all?

Reply:
Ah yes, my bad. I was first looking to catch the `-` here, so will revert this.